index.html

```diff
@@ -49,12 +49,136 @@ <h1 id="nm">I’m Mariane<br><br>
+                    <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                    <ul class="tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li class="js">javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div>
+                    <img src="./assets/Snapshoot Portfolio (2).png" alt="">
+                    <h2>Multi-Post<br>
+                        Stories</h2>
+                    <div class="details">
+                        <h3><b>CANOPY</b></h3>
+                        <ul>
+                            <li>&bull;&nbsp;&nbsp; Back End Dev</li>
+                            <li>&bull;&nbsp;&nbsp; 2015</li>
+                        </ul>
+                    </div>
+                    <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                    <ul class="tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li class="js">javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+        </section>
+        <section class="about">
+            <div>
+                <h2>About<br>Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+                    of my work and experience! If you like what you see and have a project you need coded, don’t
+                    hestiate to contact me.</p>
+                <a href="#" class="lnk">LET’S CONNECT</a>
+                <div class="socials2">
+                    <img src="./assets/Frame 1.png" alt="Socials">
+                </div>
+                <button>See my resume</button>
+
+                <div class="languages">
+                    <h2>Languages</h2>
+                    <ul>
+                        <li><img src="./assets/Ellipse 1.png" alt="" class="avatar"> Javascript</li>
+                        <li><img src="./assets/Ellipse 1 (1).png" alt="" class="avatar"> HTML</li>
+                        <li><img src="./assets/Ellipse 1 (2).png" alt="" class="avatar"> CSS</li>
+                    </ul>
+                    <div>
+                        <span class="aboutPreMenu">
+                            <span class="aboutMenu">
+                                <h3>Frameworks</h3>
+                                <img src="./assets/fwd.png" alt="Forward">
+                            </span>
+                            <hr>
+                        </span>
+                        <br>
+                        <span class="aboutPreMenu">
+                            <span class="aboutMenu">
+                                <h3>Skills</h3>
```

Comment:
- [ ] Kindly match the `Frameworks` and `Skills` headings to the `Languages` heading, the `font-size` and the `font-weight` are not the same 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/212050045-b8df359c-5b89-43d0-b31f-1a074a7cdb22.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/212050096-8e36b993-5a2c-4898-9d55-ca9d6417957d.png)